Type error on master

<!--NOTE: -->
<!--- General questions should go to the discord chat instead of the issue tracker.-->

**Describe the bug**
<!--A clear and concise description of what the bug is and steps to reproduce it.-->
There is type error present in code on master.

**Expected behavior**
<!--A clear and concise description of what you expected to happen.-->
No type errors.

**Steps to Reproduce**
<!--Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
-->
`lerna run check-types`

**Screenshots**
<!--If applicable, add screenshots to help explain your problem.-->
```
@chainsafe/lodestar: test/unit/sync/regular/oneRangeAhead/oneRangeAhead.test.ts:77:7 - error TS2345: Argument of type '(blocks: import("/home/mpetrunic/projects/eth2/lodestar/packages/lodestar-types/lib/types/block").SignedBeaconBlock[], signal: import("/home/mpetrunic/projects/eth2/lodestar/node_modules/abort-controller/dist/abort-controller").AbortSignal) => Promise<...>' is not assignable to parameter of type '(blocks: import("/home/mpetrunic/projects/eth2/lodestar/packages/lodestar-types/src/types/block").SignedBeaconBlock[], signal: AbortSignal) => Promise<void>'.
@chainsafe/lodestar:   Types of parameters 'signal' and 'signal' are incompatible.
@chainsafe/lodestar:     Type 'AbortSignal' is not assignable to type 'import("/home/mpetrunic/projects/eth2/lodestar/node_modules/abort-controller/dist/abort-controller").AbortSignal'.
@chainsafe/lodestar:       Types of property 'dispatchEvent' are incompatible.
@chainsafe/lodestar:         Type '(event: Event) => boolean' is not assignable to type '<TEventType extends string>(event: EventData<Events, TEventType, "loose">) => boolean'.
@chainsafe/lodestar:           Types of parameters 'event' and 'event' are incompatible.
@chainsafe/lodestar:             Type 'EventData<Events, TEventType, "loose">' is not assignable to type 'Event'.
@chainsafe/lodestar:               Type 'Event | NonStandardEvent | (Pick<Events[TEventType], Exclude<keyof Events[TEventType], "target" | "currentTarget" | ... 18 more ... | "timeStamp">> & Partial<...>)' is not assignable to type 'Event'.
@chainsafe/lodestar:                 Type 'NonStandardEvent' is not assignable to type 'Event'.
@chainsafe/lodestar:                   Type 'Event | NonStandardEvent' is not assignable to type 'Event'.
@chainsafe/lodestar:                     Type 'NonStandardEvent' is missing the following properties from type 'Event': bubbles, cancelBubble, cancelable, composed, and 17 more.
@chainsafe/lodestar: 77       async (blocks: SignedBeaconBlock[], signal: AbortSignal): Promise<void> => {
@chainsafe/lodestar:          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@chainsafe/lodestar: Found 1 error.
@chainsafe/lodestar: error Command failed with exit code 1.
```

**Desktop (please complete the following information):**
 - OS: <!--[e.g. ubuntu, OSX High Siera]-->
 - Version: <!--[e.g. 22]-->
 - Branch: <!--[Master]-->
 - Commit hash: <!--[e8232]-->

cc @tuyennhv  seems like it's related to new regular sync

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -29,6 +29,8 @@ jobs:
       # </common-build>
       - name: Lint
         run: yarn lint
+      - name: Check Types
+        run: yarn run check-types
       - name: Unit tests
         run: yarn test:unit
       - name: Upload coverage data

--- a/package.json
+++ b/package.json
@@ -16,6 +16,7 @@
     "build:types:watch": "lerna run build:types:watch --parallel",
     "build:watch": "run-p build:lib:watch build:types:watch",
     "lint": "lerna run lint --no-bail",
+    "check-types": "lerna run check-types --no-bail",
     "coverage": "lerna run coverage --no-bail",
     "test:unit": "lerna run test:unit --no-bail",
     "test:e2e": "lerna run test:e2e --no-bail",

--- a/packages/lodestar/src/sync/regular/oneRangeAhead/interface.ts
+++ b/packages/lodestar/src/sync/regular/oneRangeAhead/interface.ts
@@ -1,6 +1,7 @@
-import {SignedBeaconBlock} from "@chainsafe/lodestar-types/src";
+import {SignedBeaconBlock} from "@chainsafe/lodestar-types";
 import {IRegularSyncModules, ISyncCheckpoint} from "../..";
 import {IService} from "../../../node";
+import {AbortSignal} from "abort-controller";
 
 export interface IBlockRangeFetcher {
   setLastProcessedBlock(lastProcessedBlock: ISyncCheckpoint): void;

